core/server/src/main/java/alluxio/master/file/meta/InodeFile.java

```diff
@@ -340,4 +369,12 @@ public JournalEntry toJournalEntry() {
   public long getTtl() {
     return mTtl;
   }
+
+  /**
+   * @return the action to take when Ttl is expired
```

Comment:
what I meant is that every setTtlAction() method should look like 

```
/**
  * @param ttlAction the {@link TtlAction} use
  * @return ...
  */
public ... setTtlAction(TtlAction ttlAction) {
  mTtlAction = ttlAction;
  return getThis();
}
```

and every getTtlAction() method should look like this:

```
/**
  * @return the {@link PTtlAction}
  */
public TtlAction getTtlAction() {
  return mTtlAction;
}
```